feature-detail/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
**#question**

Can we specify a generic rule in our .gitignore to ignore all `**/build/**` files?